nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+  end
+end
```

Comment:
### Although I am highlighting this file, these 3 changes should be carried out in the `person.rb` file:

- [ ] The `Person` class should inherit from the `Nameable` class since it is a subset of the `Nameable` class that can be passed as a parameter to an instance of the `Base Decorator` class. Kindly implement this inheritance. 👍 
- [ ] Kindly create a `correct_name` method in the Person class that returns the @name instance variable. This will avoid errors to be thrown when testing the decorators that you created. 💻 🖥️ 
- [ ] When I tested your code, I got this error:

![image](https://github.com/KAYEMBAJONATHAN/school-library/assets/77106962/3a481696-922f-412b-b079-59a7b6c9dd10)

#### You are using Ruby's `SecureRandom` class to create the `id` for every instance of the `Person` class but it was not required before use. This will cause the error shown above. I would advise that you require at the top of the person.rb file like this; `require 'securerandom'`. 😎 

### Project Requirement

![image](https://github.com/KAYEMBAJONATHAN/school-library/assets/77106962/84e605aa-25fe-404a-932f-ef9a587b1c81)